lib/elixir_search_extractor/file_upload.ex

```diff
@@ -0,0 +1,89 @@
+defmodule ElixirSearchExtractor.FileUpload do
+  import Ecto.Query, warn: false
+
+  alias ElixirSearchExtractor.FileUpload.KeywordFile
+  alias ElixirSearchExtractor.Repo
+
+  def list_keyword_files do
+    Repo.all(KeywordFile)
```

Comment:
Should we list only keyword file that belongs to the user? 🙌

Reply:
@hanam1ni I did these changes with pagination in PR #36